project.clj

```diff
@@ -8,9 +8,9 @@
             :distribution :repo
             :comments     "same as Clojure"}
   :plugins [[lein-parent "0.3.7"]]
-  :parent-project {:coords  [io.jesi/parent "3.3.0"]
+  :parent-project {:coords  [io.jesi/parent "3.9.0"]
                    :inherit [:plugins :managed-dependencies :deploy-repositories :dependencies :profiles :test-refresh :aliases :codox]}
-  :managed-dependencies [[io.jesi/backpack "4.2.0"]]        ;TODO use manged version once 5.0 is released
+  :managed-dependencies [[io.jesi/backpack "5.2.0"]]
```

Comment:
Does backpack need backpack? I think this can be removed now

Reply:
Things that backpack uses use backpack e.g. customs